spec/spec_helper.rb

```diff
@@ -26,7 +26,8 @@
 
   config.before(:each) do
     # Mock appliance version and login api requests, as well as loading trusted certs
-    allow_any_instance_of(OneviewSDK::Client).to receive(:appliance_api_version).and_return(2000)
+
+    allow_any_instance_of(OneviewSDK::Client).to receive(:appliance_api_version).and_return(2200)
```

Comment:
any reason for changing the api_version to 2200.  Because this version is recently updated as part of our config PR

Reply:
yeah because all UTs will check all ots version to be lesser than 2200. so in case of 2000, it will not throw an error.